Woof/Woof/NetworkLayer/Endpoint/WoofAppEndpoint+Path.swift

```diff
@@ -1,6 +1,8 @@
 extension WoofAppEndpoint {
     /// Represents endpoint paths for the application.
     enum Path {
+        /// The path for getting all saved sitters.
```

Comment:
Can we get non-saved sitters?
Nit:
```suggestion
        /// The path to get all sitters.
```